examples/lock-app/efr32/src/EventHandler.cpp

```diff
@@ -0,0 +1,204 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "EventHandler.h"
+
+#if defined(ENABLE_CHIP_SHELL)
```

Comment:
Why this? Build already is:

```
if (chip_build_libshell) {
```

lets not add that many ifdefs.

Reply:
Also consider naming this EventHandlerLibShell or something like that, to make it clear this is intended for shell.

Reply:
Named this EventHandlerLibShell, and removed the ifdef.